lib/tasks/dell_demo.rake

```diff
@@ -309,7 +309,7 @@ namespace :dell_demo do
           break
         end
 
-        if input.to_i != 0
+        if input.size != 36
```

Comment:
The condition above says `==` and this one says `!=`. However, both original conditions were `!= 0`. Is this correct/intentional?

Reply:
Good catch. This was a typo. I've fixed the condition to use `==`